### PR TITLE
document id generation methods

### PR DESCRIPTION
**Pull request by nikolasburk**

### CODE REVIEW

**Review by mhessdev — DISMISSED** [review]

**Review by ankur-arch — DISMISSED** [review]

File: content/200-orm/500-reference/100-prisma-schema-reference.mdx [hunk 1]
```diff
@@ -3037,35 +3040,127 @@ model User {
...
+- For **relational databases**: If you do not want to use Prisma ORM's `uuid()` function, you can use [the native database function with `dbgenerated`](#override-default-value-behavior-for-supported-types).
+- For **MongoDB**: `uuid()` does not generate a valid `ObjectId`. You can use [`@db.ObjectId` syntax](#generate-objectid-as-ids-mongodb-only) if you want to use `ObjectId` in the underlying database. However, you can still use `uuid()` if your `_id` field is not of type `ObjectId`.
 
-<Admonition type='info'>
+#### Examples
 
-**Note (Relational databases)**: If you do not want to use Prisma ORM's `uuid()` function, you can use [the native database function with `dbgenerated`](#override-default-value-behavior-for-supported-types).
+##### Generate `uuid()` values as IDs using UUID v4
```
Comment (jharrell) [inline content/200-orm/500-reference/100-prisma-schema-reference.mdx:3098]: nit: you removed the parens `()` above, but kept them here and below. Could we standardize?

File: content/200-orm/500-reference/100-prisma-schema-reference.mdx [hunk 2]
```diff
@@ -3000,19 +3004,18 @@ model User {
...
-
-##### MongoDB
-
-- `cuid()` does not generate a valid `ObjectId` - [use the `@db.ObjectId` syntax](#generate-objectid-as-ids-mongodb-only) if you want to use `ObjectId` in the underlying database. However, you can still use `cuid()` if your `_id` field is not of type `ObjectId`.
+- For **MongoDB**: `cuid()` does not generate a valid `ObjectId`. You can use [`@db.ObjectId` syntax](#generate-objectid-as-ids-mongodb-only) if you want to use `ObjectId` in the underlying database. However, you can still use `cuid()` if your `_id` field is not of type `ObjectId`.
 
 #### Examples
 
-##### Generate `cuid()` values as IDs
+##### Generate `cuid` values as IDs
```
Comment (nikolasburk) [inline content/200-orm/500-reference/100-prisma-schema-reference.mdx]:
```suggestion
##### Generate `cuid()` values as IDs
```

File: content/200-orm/500-reference/100-prisma-schema-reference.mdx [hunk 3]
```diff
@@ -3037,35 +3040,127 @@ model User {
 </TabItem>
 </TabbedContent>
 
+##### Generate `cuid2` values as IDs
```
Comment (nikolasburk) [inline content/200-orm/500-reference/100-prisma-schema-reference.mdx]:
```suggestion
##### Generate `cuid(2)` values as IDs based on the `cuid2` spec 
```

File: content/200-orm/500-reference/100-prisma-schema-reference.mdx [hunk 4]
```diff
@@ -3037,35 +3040,127 @@ model User {
...
+model User {
+  id   String @id @default(uuid()) @map("_id")
+  name String
+}
+```
+
+</TabItem>
+</TabbedContent>
+
+##### Generate `uuid()` values as IDs using UUID v7
```
Comment (nikolasburk) [inline content/200-orm/500-reference/100-prisma-schema-reference.mdx]:
```suggestion
##### Generate `uuid(7)` values as IDs using UUID v7
```

File: content/200-orm/500-reference/100-prisma-schema-reference.mdx (hunk 1 above)
Comment (nikolasburk) [reply content/200-orm/500-reference/100-prisma-schema-reference.mdx:3098]: nice catch! looks like we used `()` before, so I adjusted the instanced that didn't have them above 🙏

**Review by ankur-arch — APPROVED** [review]

**Pull request merged**